static/app/utils/profiling/profile/importProfile.tsx

```diff
@@ -232,7 +232,10 @@ function importSchema(
   traceID: string,
   options: ImportOptions
 ): ProfileGroup {
-  const frameIndex = createFrameIndex('mobile', input.shared.frames);
+  const frameIndex = createFrameIndex(
+    input.metadata.platform === 'node' ? 'node' : 'mobile',
```

Comment:
Does this need to aware of the other platforms like pytho and rust? If so, would it make sense to pass the platform through instead of doing this conversion here and classify everything else as `mobile`?

Reply:
Yes - I want to pass profile.metadata.platform here, but I will do that in a separate PR. Do you know if/where we have the list of platforms that the backend accepts?

Reply:
[This](https://github.com/getsentry/vroom/blob/1ec612171e235db1e89293fb776212cfce00f578/internal/profile/legacy.go#L100) has a list from the legacy format but now that we're on the sampled format, we don't have a comprehensive list as any platform implementing the format will be accepted